Data/HashMap/Base.hs

```diff
@@ -768,6 +769,19 @@ traverseWithKey f = go
         Collision h <$> A.traverse (\ (L k v) -> L k <$> f k v) ary
 {-# INLINE traverseWithKey #-}
 
+-- | /O(n)/ Map each element of a map to an action, evaluate these
+-- actions, and ignore the results.
+traverseWithKey_ :: Applicative f => (k -> v1 -> f b) -> HashMap k v1
+                -> f ()
+traverseWithKey_ f = go
+  where
+    go Empty                 = pure ()
+    go (Leaf _ (L k v))      = f k v *> pure ()
```

Comment:
Check the Travis build failure. `*>` is not in scope. Make sure this change works with the latest 3 major GHC releases.